### DISCUSSION TITLE
Adguard Home是不是属于域名解析服务，在中国国内可以用服务器搭建么

### DISCUSSION THREAD

**Discussion by hengyv1998** [AdguardTeam/AdGuardHome#4167] [Q&A]
Is adguard home a domain name resolution service that can be built with a server in China

**Comment by hacamer** [comment]
Yes，it's a dns sever. If you want , you can install it with a sever in China.

**Comment by vzchn** [comment]
搭建了不面向公网没事，面向公网要关停，并且要求签保证书。

**Comment by hengyv1998** [comment]
好的谢谢大佬












Other&nbsp;shore&nbsp;flow
***@***.***


---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年1月9日(周日) 上午10:56
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [AdguardTeam/AdGuardHome] Adguard Home是不是属于域名解析服务，在中国国内可以用服务器搭建么 (Issue #4096)




 
搭建了不面向公网没事，面向公网要关停，并且要求签保证书。
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
